pages/index.js

```diff
@@ -1,7 +1,185 @@
+import Triangle from '../components/svg/Triangle';
+import Image from 'next/image';
+import Link from 'next/link';
+import Service from '../components/Service';
+import WhatWeDo from '../components/WhatWeDo';
+import Blog from '../components/Blog-link';
+
 export default function Home() {
```

Comment:
You haven't capitalized the other page components why is this one capitalized?

Reply:
every item in my components folder is capitalized. the pages folder is all lowercase

Reply:
i made what we do and service as components to be used in any page